invenio_drafts_resources/resources/draft.py

```diff
@@ -119,8 +124,9 @@ def __init__(self, service=None, *args, **kwargs):
 
     def create(self, *args, **kwargs):
         """Any POST business logic."""
+        # FIXME: Implement as CMD patter loaded from config
         if resource_requestctx.route["action"] == "publish":
             identity = g.identity
             id_ = resource_requestctx.route["pid_value"]
-            return self.service.publish(id_, identity), 200
-        return {}, 200
+            return self.service.publish(id_, identity), 202
+        return {}, 202
```

Comment:
I would leave as 202. It is true for now it is not an async task. But once the command pattern is there for services, it will be. My point being it will be soon, as soon as we start adding more complexity to it.